src/main/java/laba/ReservationService.java

```diff
@@ -0,0 +1,209 @@
+                default:
+                    System.out.println("Invalid choice. Please try again.");
+            }
+        }
+    }
+
+    private static void customerMenu() {
+        System.out.println("Customer Menu");
+        while (true) {
+            System.out.println("1. Browse available spaces");
+            System.out.println("2. Make a reservation");
+            System.out.println("3. View my reservations");
+            System.out.println("4. Cancel a reservation");
+            System.out.println("5. Exit to main menu");
+            System.out.print("Choose an option: ");
+            int choice = SCANNER.nextInt();
+            SCANNER.nextLine();
+            switch (choice) {
+                case 1:
+                    viewCoworkingSpaces();
+                    break;
+                case 2:
+                    makeReservation();
+                    break;
+                case 3:
+                    viewMyReservations();
+                    break;
+                case 4:
+                    cancelReservation();
+                    break;
+                case 5:
+                    return;
+                default:
+                    System.out.println("Invalid choice. Please try again.");
+            }
+        }
+    }
+
+    private static void viewCoworkingSpaces() {
+        System.out.println("Available Coworking Spaces:");
+        for (CoworkingSpace space : SPACES) {
+            System.out.println(space);
+        }
+    }
+
+    private static void addCoworkingSpace() {
+        System.out.print("Enter new space type (e.g., Open Space, Private Room): ");
+        String type = SCANNER.nextLine();
+        System.out.print("Enter price per hour: ");
+        double price = SCANNER.nextDouble();
+        SCANNER.nextLine();
+        System.out.print("Is it available (true/false): ");
+        boolean isAvailable = SCANNER.nextBoolean();
+        SCANNER.nextLine();
+        CoworkingSpace newSpace = new CoworkingSpace(SPACES.size() + 1, type, price, isAvailable);
+        SPACES.add(newSpace);
+        System.out.println("Coworking space added successfully.");
+    }
+
+    private static void removeCoworkingSpace() {
+        System.out.print("Enter the ID of the space to remove: ");
+        int id = SCANNER.nextInt();
+        SCANNER.nextLine();
+        CoworkingSpace spaceToRemove = null;
+        for (CoworkingSpace space : SPACES) {
+            if (space.getId() == id) {
+                spaceToRemove = space;
+                break;
+            }
+        }
+        if (spaceToRemove != null) {
+            SPACES.remove(spaceToRemove);
+            System.out.println("Coworking space removed successfully.");
+        } else {
+            System.out.println("Space with the given ID not found.");
+        }
+    }
+
+    private static void makeReservation() {
+        viewCoworkingSpaces();
+        System.out.print("Enter the ID of the space to reserve: ");
```

Comment:
try follow SOLID principles (separate logic for admin/customer, create new classes and use instances of new classes)